backend/src/main/java/com/ody/notification/domain/Notification.java

```diff
@@ -74,7 +74,11 @@ public static Notification createDepartureReminder(Mate mate, LocalDateTime send
     }
 
     public static Notification createNudge(Mate mate) {
-        return new Notification(mate, NotificationType.NUDGE, LocalDateTime.now(), NotificationStatus.PENDING, null);
+        return new Notification(mate, NotificationType.NUDGE, LocalDateTime.now(), NotificationStatus.DONE, null);
+    }
+
+    public boolean isDepartureReminder() {
+        return this.type == NotificationType.DEPARTURE_REMINDER;
```

Comment:
[제안]
조금 더 객체지향스럽게 어떤가요?
1.
```suggestion
        return type.isSameType(NotificationType.DEPARTURE_REMINDER);
```
2.
```suggestion
        return type.isDepartureReminder();
```

Reply:
조화요 반영완 ! 👍